docs/tutorials/BuildMultiClusterApplication/BuildMultiClusterApplication.md

```diff
@@ -0,0 +1,524 @@
+      type: WorkSynchronized
+    - lastTransitionTime: "2024-11-25T09:02:31Z"
+      message: All corresponding work objects are applied
+      observedGeneration: 3
+      reason: AllWorkHaveBeenApplied
+      status: "True"
+      type: Applied
+    - lastTransitionTime: "2024-11-25T09:02:31Z"
+      message: All corresponding work objects are available
+      observedGeneration: 3
+      reason: AllWorkAreAvailable
+      status: "True"
+      type: Available
+  selectedResources:
+  - kind: Namespace
+    name: multi-cluster-app
+    version: v1
+  - group: networking.fleet.azure.com
+    kind: ServiceExport
+    name: nginx-service
+    namespace: multi-cluster-app
+    version: v1alpha1
+  - kind: Service
+    name: nginx-service
+    namespace: multi-cluster-app
+    version: v1
+  - kind: ConfigMap
+    name: envelope-configmap
+    namespace: multi-cluster-app
+    version: v1
+```
+### Validate the trafficManagerEndpoint
+
+To validate whether the service of Member Cluster 2 is exposed via Traffic Manager, you can check the status of the TrafficManagerBackend object.
+
+```bash
+kubectl get tmb nginx-backend -n multi-cluster-app -o yaml
+apiVersion: networking.fleet.azure.com/v1alpha1
+kind: TrafficManagerBackend
+metadata:
+  annotations:
+    kubectl.kubernetes.io/last-applied-configuration: |
+      {"apiVersion":"networking.fleet.azure.com/v1alpha1","kind":"TrafficManagerBackend","metadata":{"annotations":{},"name":"nginx-backend","namespace":"multi-cluster-app"},"spec":{"backend":{"name":"nginx-service"},"profile":{"name":"nginx-profile"},"weight":100}}
+  creationTimestamp: "2024-11-25T08:49:09Z"
+  finalizers:
+  - networking.fleet.azure.com/traffic-manager-backend-cleanup
+  generation: 1
+  name: nginx-backend
+  namespace: multi-cluster-app
+  resourceVersion: "3718954"
+  uid: 90089f42-222a-4d9a-8a97-8b5ed2f9ca42
+spec:
+  backend:
+    name: nginx-service
+  profile:
+    name: nginx-profile
+  weight: 100
+status:
+  conditions:
+  - lastTransitionTime: "2024-11-25T09:02:56Z"
+    message: 2 service(s) exported from clusters have been accepted as Traffic Manager
+      endpoints
+    observedGeneration: 1
+    reason: Accepted
+    status: "True"
+    type: Accepted
+  endpoints:
+  - cluster:
+      cluster: member-1
+    name: fleet-90089f42-222a-4d9a-8a97-8b5ed2f9ca42#nginx-service#member-1
+    target: fleet-multi-cluster-app-member-1.westcentralus.cloudapp.azure.com
+    weight: 50
+  - cluster:
+      cluster: member-2
+    name: fleet-90089f42-222a-4d9a-8a97-8b5ed2f9ca42#nginx-service#member-2
+    target: fleet-multi-cluster-app-member-2.westcentralus.cloudapp.azure.com
+    weight: 50
+```
+Summary:
+* The traffic is now split between the two clusters, member-1 and member-2, with a weight of 50% each.
+* Similarly, when the cluster is left from the fleet by deleting memberCluster CR, all the placed resources will be removed from the cluster, and it
+will be removed from the Azure Traffic Manager automatically.

```

Comment:
so what actually will happen to the traffic when a cluster leaves the fleet? It seems that it will still go there?

Reply:
Today the networking resources will be deleted when the cluster left as the serviceExport/mcs does not have any meaning when the cluster is not part of the fleet any more.

When a cluster is left, all the workloads (excluding serviceExport) are kept in the member cluster.

The new traffic will go to other cluster and old traffic can be served by the old cluster.

Let me validate my understanding.